tasks/localizationTasks.ts

```diff
@@ -0,0 +1,174 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as gulp from 'gulp';
+import * as process from 'node:process';
+import * as minimist from 'minimist';
+import { spawnSync } from 'node:child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as util from 'node:util';
+import { EOL } from 'node:os';
+import { Octokit } from '@octokit/rest';
+
+type Options = {
+    userName: string;
+    email: string;
+    owner: string;
```

Comment:
I think owner is the fork owner right?
Is it necessary to support forks?  IMHO I'd prefer keeping the number of options down and just always push to the main repo unless there is a really good reason not to

Reply:
Ah you mean hardcode `dotnet` here? Sure it can be done